addons/pharma/XEH_postInit.sqf

```diff
@@ -18,4 +18,40 @@
 
 ["kat_PainkillerItem", "kat_Painkiller"] call ACEFUNC(common,registerItemReplacement);
 ["kat_CarbonateItem", "kat_Carbonate"] call ACEFUNC(common,registerItemReplacement);
-["kat_PervitinItem", "kat_Pervitin"] call ACEFUNC(common,registerItemReplacement);
\ No newline at end of file
+["kat_PervitinItem", "kat_Pervitin"] call ACEFUNC(common,registerItemReplacement);
+
+
+["ace_treatmentSucceded", {
+    params ["_medic", "_patient", "_bodyPart", "_classname"];
+    if (!local _patient) exitWith {["ace_treatmentSucceded", _this, _patient] call CBA_fnc_targetEvent};
+    if (_classname == "Epinephrine") then {
+        [_medic, _patient] call ace_medical_treatment_fnc_cprSuccess;
+        _patient setVariable [QGVAR(Epinephrine), 1, true];
+
+		if (ACEGVAR(advanced_fatigue,enabled)) then {
+			params ["_patient"];
+			ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 1000;
```

Comment:
i am not a fan of touching those gvars as Baer said but we will see in future

Reply:
I think baer ment changing things like the add-on settings mid game, like making the recovery factor better. This is only giving you a stamina reserve so I think it should be good.